src/Symfony/Component/Translation/Catalogue/AbstractOperation.php

```diff
@@ -150,9 +150,13 @@ public function getObsoleteMessages($domain)
     /**
      * {@inheritdoc}
      */
-    public function getResult()
+    public function getResult($domain = null)
```

Comment:
I'm not comfortable with this change. The $operation should already have filtered domains.
I see two alternatives: add `setDomains()` on `AbstractOperation`, and add a `$domains` argument to `MessageCatalogue` to make it filter messages it can contains. A third alt would be to filter at the loader/extractor level, but they look too specific for this. WDYT?